test/features/complexTypes.feature

```diff
@@ -1,7 +1,42 @@
+      | Child       | ChildAddress                     | AddressUK   |                    | Child address                        |
+      | Child       | IsAutistic                       | YesOrNo     |                    | Is the child autistic?               |
+      | Child       | NeedsSupport                     | YesOrNo     |                    | Does the child needs support?        |
+      | Child       | AutisticChildCaseNumber          | CaseLink    |                    | Autistic child case number reference |
+
+    Given the following definition for 'complex case fields overrides'
+      | Id       | Path                                                              | Display context | Label                     | Hint text            | Field show condition                            |
+      | MySchool | Name                                                              | MANDATORY       |                           |                      |                                                 |
+      | MySchool | ProvidesAutisticChildrenSupport                                   | MANDATORY       |                           |                      |                                                 |
+      | MySchool | Class.ClassName                                                   | MANDATORY       |                           |                      |                                                 |
+      | MySchool | Class.ClassNumber                                                 | OPTIONAL        |                           |                      | MySchool.Class.ClassName="A team"               |
+      | MySchool | Class.ClassMembers.Children.IsAutistic                            | OPTIONAL        |                           |                      | MySchool.ProvidesAutisticChildrenSupport="Yes"  |
+      | MySchool | Class.ClassMembers.Children.ChildFullName                         | MANDATORY       | Child full name (UPDATED) | Child hint (UPDATED) |                                                 |
+      | MySchool | Class.ClassMembers.Children.ChildGender                           | MANDATORY       |                           |                      |                                                 |
+      | MySchool | Class.ClassMembers.Children.ChildAddress.AddressLine1             | OPTIONAL        |                           |                      |                                                 |
+      | MySchool | Class.ClassMembers.Children.NeedsSupport                          | OPTIONAL        |                           |                      |                                                 |
+      | MySchool | Class.ClassMembers.Children.AutisticChildCaseNumber.CaseReference | OPTIONAL        |                           |                      |                                                 |
     Given I have logged in
 
   Scenario: limits the complex type using the EventToComplexTypes sheet
```

Comment:
can you change `limits the complex type using the EventToComplexTypes sheet` to `limits the complex type using the Event to Complex Type override feature` please

Reply:
Will change to:
Scenario: filters the complex type fields using the Event to Complex Type override feature
Scenario: overrides the complex type data using the Event to Complex Type override feature